ES/ESmy020.xml

```diff
@@ -31,71 +31,394 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
-                              <item xml:id="q6">
-                                 <locus from="44r" to="53v"/> VI(fols. 44r-53v) </item>
-                              <item xml:id="q7">
-                                 <locus from="54r" to="71v"/> VII(fols. 54r-71v)&#xD; &#xD; Quire I
-                                 is composed entirely of fols. from other parts of the Ms., its
-                                 folios are attached as guard leaves by means of crude stitches. All
-                                 the folios of quire I belong to Text I.</item>
+                              <item xml:id="q1" n="1">
+                                 <locus from="1r" to="5v"/>
+                                 <dim unit="leaf">4</dim>
+                                  <locus from="1r" to="14v"/>
+                                  s.l.: 1, no stub
+                                  s.l.: 2, no stub
+                                  s.l.: 3, no stub
+                                  s.l.: 4, no stub
+                                   </item>
+                              <item xml:id="q2" n="2">
+                                 <locus from="6r" to="7v"/>
+                                 <dim unit="leaf">2</dim>
+                                  <dim unit="leaf">2</dim>
+                                </item>
+                              <item xml:id="q3" n="3">
+                                 <locus from="6r" to="7v"/>
+                                 <dim unit="leaf">2</dim>>
+                                 <locus from="15r" to="22v"/>
+                                 <dim unit="leaf">8</dim>
+                               </item>
+                              <item xml:id="q4" n="4">
+                                 <locus from="23r" to="33v"/>
+                                  <dim unit="leaf">11</dim>
+                                  s.l.: 11, stub after 10
+                                 </item>
+                              <item xml:id="q5" n="5">
+                                 <locus from="34r" to="43v"/>
+                                 <dim unit="leaf">10</dim>
+                               </item>
+                              <item xml:id="q6" n="6">
+                                 <locus from="44r" to="53v"/>
+                                 <dim unit="leaf">10</dim>
+                                 </item>
+                              <item xml:id="q7" n="7">
+                                 <locus from="54r" to="63v"/>
+                                 <dim unit="leaf">10</dim>
+                                  </item>
+
+                                  <item xml:id="q8" n="8">
+                                     <locus from="64r" to="71v"/>
+                                     <dim unit="leaf">8</dim>
+                                     <dim unit="leaf">10</dim>
+                                     s.l.: 5, stub after 4
```

Comment:
why second dim element?
the quire composition does not work again

Reply:
must be fixed or won't be visible on the app

Reply:
corrected